container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -276,6 +276,16 @@ if [ ! -f ${TEMPEST_PATH}exclude.txt ] && [ -z ${TEMPEST_EXCLUDE_LIST} ]; then
     touch ${TEMPEST_PATH}exclude.txt
 fi
 
+# This workaround is required for the whitebox-neutron-tempest plugin. We need
+# to be able to specify 600 permissions for the id_ecdsa.
+if [ -f ${HOMEDIR}/id_ecdsa ]; then
+    mkdir ${HOMEDIR}/.ssh
+    cp ${HOMEDIR}/id_ecdsa ${HOMEDIR}/.ssh/id_ecdsa
```

Comment:
Thank you for the changes, looks good.
Only on my setup deployed before these changes, it seem 'id_ecdsa' was root owned, so maybe it would need 'sudo cp ...' too.
`
sh-5.1$ ls -lh /var/lib/tempest/.ssh/id_ecdsa
-rw-r--r--. 1 root tempest 512 Apr  4 10:12 /var/lib/tempest/.ssh/id_ecdsa
`

Reply:
It should not be a problem, shouldn't it? The file is owned by the tempest group with read access.

Reply:
Right, that would work too if mkdir runs as tempest user.
All looks good to me, thanks.